packet/packet.py

```diff
@@ -48,9 +50,11 @@ def get_signatures(freshman_username):
             'misc': misc_signatures}
 
 
+@lru_cache(maxsize=512)
```

Comment:
I'm a bit wary of caching these so simply. I think we will end up with stale values very quickly. We should at least clear the cache when a packet is signed.